source/winConsoleHandler.py

```diff
@@ -109,7 +109,7 @@ def disconnectConsole():
 	# Unregister any win events we are using
 	for handle in consoleWinEventHookHandles:
 		winUser.unhookWinEvent(handle)
-	consoleEventHookHandles = []  # noqa: F841
+	consoleWinEventHookHandles = []
```

Comment:
@michaelDCurran, @codeofdusk or @jcsteh:
Before this change, the list `consoleWinEventHookHandles` was not cleared. This means that we were potentially calling `winUser.unhookWinEvent` on handles of events already unhooked. Could this have caused any noticeable bug?